src/zip/Zip.js

```diff
@@ -136,6 +136,16 @@ class Zip {
         }
     }
 
+    removeFile = (fileName) => {
+        try {
+            Utils.log(`Remove file: ${fileName}`)
+            const fileIndex = this.fileRecord.findIndex(record => record.name === fileName)
+            if (fileIndex !== -1) this.fileRecord.splice(fileIndex, 1)
+        } catch (e) {
+            Utils.error(e)
+        }

```

Comment:
What do you think should be expected to fail in this block?

Reply:
hmm, nothing. I'll remove it. 😁